src/arbitrum/epoch-based-managed-vaults/stratagies/interfaces/IY2KVaultFactory.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+interface IY2KVaultFactory {
+
+    /// @dev Function the retrieve the addresses of the hedge and risk vaults, in an array, in the respective order
+    /// @param _index - uint256 of the market index which to the vaults are associated to
+    /// @return _vaults - Address array of two vaults addresses, [0] being the hedge vault, [1] being the risk vault
+    function getVaults(uint256 _index) external returns (address[] memory _vaults);
```

Comment:
Not sure if there are any weird side effects of this but if we know there are only 2 items can we maybe safe gas here? 

```suggestion
    function getVaults(uint256 _index) external returns (address[2] memory _vaults);
```

Reply:
The function returns a dynamic array. See (https://arbiscan.io/address/0x984e0eb8fb687afa53fc8b33e12e04967560e092#code#F1#L602).